apps/dashboard/src/components/ipfs-upload/dropzone.tsx

```diff
@@ -74,12 +74,13 @@ export const IpfsUploadDropzone: React.FC = () => {
   });
   return (
     <Flex flexDir="column" gap={4}>
-      <AspectRatio
-        ratio={{
-          base: droppedFiles.length ? 1 : 8 / 4,
-          md: droppedFiles.length ? 16 / 9 : 36 / 9,
-        }}
-        w="100%"
+      <div
+        className={cn(
+          "w-full",
+          droppedFiles.length
+            ? "aspect-square md:aspect-[16/9]"
+            : "aspect-[4/1] md:aspect-[36/9]",
```

Comment:
should be `8/4` for `base` no? which ... isn't that just `2` then?

Reply:
no way I just failed math

Reply:
omg